platform/legacy/LegacyPlatformProxifier.ts

```diff
@@ -0,0 +1,87 @@
+import { Server } from 'net';
+import { IncomingMessage, OutgoingMessage } from 'http';
+import { EventEmitter } from 'events';
+
+import { Logger } from '../logging';
+
+export class LegacyPlatformProxifier extends EventEmitter {
+  private server: Server;
+  private readonly eventHandlers: any;
+
+  constructor(
+    private readonly log: Logger,
+    readonly kbnServer: any,
+    private readonly startPlatform: () => Promise<any>,
+    private readonly stopPlatform: () => Promise<any>
+  ) {
+    super();
+
+    const forwardEvent = (name: string, ...args: any[]) => {
```

Comment:
niiiitpick:

```ts
const forwardEvent = (name: string) => (...args: any[]) => {
  this.log.info(`Event is being forwarded: ${name}`);
  this.emit(name, ...args);
};
```

then there no need to `.bind` below

Reply:
✔️